### PR TITLE
Refactor WebRTC class

### PR DESCRIPTION
**Pull request by ggazzo**

### CODE REVIEW

File: app/webrtc/client/WebRTCClass.js [hunk 1]
```diff
@@ -165,7 +129,7 @@ class WebRTCClass {
 		this.config = {
 			iceServers: [],
 		};
-		this.debug = false;
+		this.debug = true;
```
Comment (wreiske) [inline app/webrtc/client/WebRTCClass.js]: Should this be true?

Comment (ggazzo) [reply app/webrtc/client/WebRTCClass.js]: wip ;)

Comment (ggazzo) [reply app/webrtc/client/WebRTCClass.js]: but thanks

File: app/notifications/server/lib/Notifications.js [hunk 2]
```diff
@@ -175,7 +177,7 @@ const notifications = new Notifications();
 notifications.streamRoom.allowWrite(function(eventName, username, typing, extraData) {
 	const [roomId, e] = eventName.split('/');
 
-	if (e === 'webrtc') {
+	if (e === WEB_RTC_EVENTS.WEB_RTC) {
```
Comment (rodrigok) [inline app/notifications/server/lib/Notifications.js]: How can this work? `'0' === 0`?

File: app/notifications/server/lib/Notifications.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 import { Meteor } from 'meteor/meteor';
 import { DDPCommon } from 'meteor/ddp-common';
+
+import { WEB_RTC_EVENTS } from '../../../webrtc';
```
Comment (rodrigok) [inline app/notifications/server/lib/Notifications.js]: This index file doesn't exists anymore

Comment (ggazzo) [reply app/notifications/server/lib/Notifications.js]: are you sure?

**Comment by ggazzo** [comment]
@rodrigok could you review before the next conflict?

**Pull request merged**